dspace-spring-rest/src/test/java/org/dspace/app/rest/matcher/SubmissionFormFieldMatcher.java

```diff
@@ -87,4 +89,38 @@ private SubmissionFormFieldMatcher() { }
             hasJsonPath("$.hints", containsString(hints))
         );
     }
+
+    public static Matcher<? super Object> matchFormOpenRelationshipFieldDefinition(String type, String label,
+                                                                                   String mandatoryMessage,
+                                                                                   boolean repeatable, String hints,
+                                                                                   String metadata,
+                                                                                   String relationshipType,
+                                                                                   String filter,
+                                                                                   String searchConfiguration) {
+        return allOf(
+            hasJsonPath("$.selectableRelationships[0].relationshipType", is(relationshipType)),
+            hasJsonPath("$.selectableRelationships[0].filter", is(filter)),
+            hasJsonPath("$.selectableRelationships[0].searchConfiguration", is(searchConfiguration)),
+            matchFormFieldDefinition(type, label, mandatoryMessage, repeatable, hints, metadata));
+    }
+
+    public static Matcher<? super Object> matchFormClosedRelationshipFieldDefinition(String label,
```

Comment:
Please add Javadocs, just like other matchers. I think this is especially important here because the phrase "closed relationship" isn't defined anywhere in this code... so this method name isn't self-explanatory

Reply:
The JavaDocs have been included